docs/pages/management/admin/self-signed-certs.mdx

```diff
@@ -0,0 +1,216 @@
+  - `auth_servers: ["tele.example.com:443"]` or
+  - `auth_servers: ["tele.example.com:3080"]`
+- Teleport is started with the `--auth-server` flag pointing to the Proxy Service endpoint:
+  - `teleport [app | db] start --auth-server=tele.example.com:443` or
+  - `teleport [app | db] start --auth-server=tele.example.com:3080`
+
+The way you make your Teleport service run with TLS certificate verification disabled depends on how you are
+running it: via the `teleport` CLI, using a Helm chart, or via systemd:
+<Tabs>
+
+    <TabItem label="Using the CLI">
+    When running `teleport` from the command line, pass the `--insecure` flag to disable
+    TLS certificate validation. For example:
+    ```sh
+    $ teleport start -c /etc/teleport.yaml --insecure
+    $ teleport app start -c /etc/teleport.yaml --insecure
+    $ teleport db start -c /etc/teleport.yaml --insecure
+    ```
+    Without the `--insecure` flag, you will see an error message that looks like
+    `x509: “tele.example.com” certificate is not trusted`.
+    </TabItem>
+
+    <TabItem label="Helm chart">
+    For deploying a cluster, set
+    [extraArgs](../../reference/helm-reference/teleport-cluster.mdx#extraargs)
+    to include the extra argument: `--insecure`:
+    <Tabs>
+      <TabItem label="values.yaml">
+      ```yaml
+      extraArgs:
+      - "--insecure"
+      ```
+      </TabItem>
+      <TabItem label="--set">
+      ```code
+      $ --set "extraArgs={--insecure}"
+      ```
+      </TabItem>
+    </Tabs>
+    
+    
+    For a Helm Kube Agent, set the 
+    [insecureSkipProxyTLSVerify
+    ](../../reference/helm-reference/teleport-kube-agent.mdx#insecureskipproxytlsverify)
+    flag to `true`:
+    <Tabs>
+      <TabItem label="values.yaml">
+      ```yaml
+      insecureSkipProxyTLSVerify: true
+      ```
+      </TabItem>
+      <TabItem label="--set">
+      ```code
+      $ --set insecureSkipProxyTLSVerify=true
+      ```
+      </TabItem>
+    </Tabs>
+    </TabItem>
+
+    <TabItem label="systemd">
+    Edit the `systemd` Teleport unit file to include `--insecure` in the `ExecStart` line, for example:
+    
+    ```text
+    ExecStart=/usr/local/bin/teleport start --pid-file=/run/teleport.pid --insecure
+    ```
+    
+    You also need to reload the daemon for your changes to take effect:
+    ```sh
+    $ sudo systemctl daemon-reload
+    $ sudo systemctl restart teleport.service
+    ```
+    
+    See [create a systemd service](./daemon.mdx#step-23-create-and-configure-a-systemd-service) for
+    more information.
+    </TabItem>
+
+</Tabs>
+
+### `tctl`
+When running `tctl` remotely via the Teleport Proxy Service, if the Proxy Service is using self-signed 
+certificates, then `tctl` will not trust the certificate from the Proxy Service.
+To disable certificate verification use:
+`tctl --insecure`.
+`tctl` will determine how to connect to the auth service in a few ways:
+- loading configuration from the profile after logging in with `tsh`
```

Comment:
```suggestion
- loading configuration from a local profile after logging in with `tsh`
```
Since we haven't mentioned "profile" yet, I've suggested adding an indefinite article and qualifying this a bit more.